fuse_graphs/CMakeLists.txt

```diff
@@ -28,7 +28,28 @@ catkin_package(
 ###########
 ## Build ##
 ###########
-add_compile_options(-Wall -Werror)
+
+# Disable warnings about maybe uninitialized variables with -Wno-maybe-uninitialized until we fix the following error:
```

Comment:
@ayrton04 For this one though, I don't know if there's any issue already reported in Boost.

The thing is that I'm not even sure if the warning is caused by `boost::iterators` or the lamdba we're using. :thinking: